tests/plugins/kernel_install/arch_test.sh

```diff
@@ -3,7 +3,7 @@
 . ./src/plugins/kernel_install/utils.sh --source-only
```

Comment:
apparently I forgot about the plugin tests when implementing include. Maybe another commit for this could be a good idea (or we should open another issue)